qualtran/bloqs/arithmetic/comparison.py

```diff
@@ -947,6 +948,78 @@ def _gt_k() -> GreaterThanConstant:
 _GREATER_THAN_K_DOC = BloqDocSpec(bloq_cls=GreaterThanConstant, examples=[_gt_k])
 
 
+@frozen
+class Equals(Bloq):
+    r"""Implements |x>|y>|t> => |x>|y>|t ⨁ (x = y)> using $n-1$ Toffoli gates.
+    Args:
```

Comment:
add empty lines between the different sections here and below
```suggestion

    Args:
```